content/en/bits_ai/bits_ai_sre/investigate_issues.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Investigate issues
+description: "Use Bits AI SRE to automatically investigate monitor alerts and provide root cause analysis for faster incident resolution."
+aliases: 
+- /bits_ai/bits_ai_sre/investigate_alerts/
+---
+
+## Get started with alert investigations
+
+
+The starting points of a Bits AI SRE investigation are:
+- Monitor alerts, which you can trigger in two ways:
+  - [**Manual**](#manual-monitor-alerts): Start from an individual monitor alert, APM latency graph, or APM Watchdog story
+  - [**Automatic**](#enable-automatic-investigations): Configure monitors so that whenever they alert, Bits launches an investigation
+- [APM latency graphs on service pages](#apm-latency-graphs-on-service-pages)
+- [APM latency Watchdog stories](#apm-latency-watchdog-stories)
```

Comment:
This list here is a little conceptually odd to me vis-a-vis the layout of the page. It seems like rather than the current hierarchy of concepts, it's more like

- Manual
   - Monitor alerts
   - APM latency graphs
   - APM latency Watchdog stories
- Automatic
   - Monitor alerts again